projectmanagement/src/main/java/life/qbic/projectmanagement/application/ProjectCreationService.java

```diff
@@ -43,15 +52,22 @@ public ProjectCreationService(ProjectRepository projectRepository) {
    */
   public Result<Project, ApplicationException> createProject(String code, String title, String objective,
       String experimentalDesign, String sourceOffer, PersonReference projectManager,
-      PersonReference principalInvestigator, PersonReference responsiblePerson) {
+      PersonReference principalInvestigator, PersonReference responsiblePerson,
+      List<Species> speciesList, List<Analyte> analyteList, List<Specimen> specimenList) {
+
     try {
-      Project project;
-      project = createProject(code, title, objective, experimentalDesign,
+      Project project = createProject(code, title, objective, experimentalDesign,
           projectManager, principalInvestigator, responsiblePerson);
+      projectRepository.add(project);
       Optional.ofNullable(sourceOffer)
           .flatMap(it -> it.isBlank() ? Optional.empty() : Optional.of(it))
           .ifPresent(offerIdentifier -> project.linkOffer(OfferIdentifier.of(offerIdentifier)));
-      projectRepository.add(project);
+
+      Experiment experiment = Experiment.createForProject(project, analyteList, specimenList,
```

Comment:
Why passing the complete project entity to the experiment? I think the referenceId should be enough and will also avoid leaking dependencies. I would also not use `createForProject`, because in the domain it does not make sense. You define an experiment or plan it. And then later you conduct it. 

I also dont understand, why the analyteList, specimentList and speciesList sneaks into the experiment. Can you explain?

Reply:
When we discussed this, I got the information that those are properties of the experiment not the project. This is why they are in the experiment.
We pass the whole project for jpa to do the mapping correctly.

Reply:
Hi @KochTobi , after talking back to Jennifer I think we should just provide the minimal information on creation, meaning passing the project-reference-id and the name of the experiment. Everything else is mutable and can be manipulated via the Experiments API, like `addSpecies(String... species)` etc. What do you think?

Reply:
I can change that :+1: